### PR TITLE
update VOs on NCG-INGRID-PT

### PR DESCRIPTION
**Pull request by mariojmdavid**
I have update the VOs supported at NCG-INGRID-PT

### CODE REVIEW

**Comment by enolfc** [comment]
Thanks @mariojmdavid!
While you created the PR I merged one that seems to introduce a conflict here, for some reason I added training.egi.eu VO there, can you check if that makes sense?

**Comment by mariojmdavid** [comment]
the training VO is correct to add, and the projectID also correct, you can solve this conflict

File: sites/NCG-INGRID-PT.yaml [hunk 1]
```diff
@@ -7,6 +7,15 @@ vos:
...
+  opencoast.eosc-hub.eu:
+    auth:
+      project_id: b0cea6bd85844b0693ceda70d9f94a09
+  worsica.vo.incd.pt:
+    auth:
+      project_id: a53ca78c534046e5b13f4537ae698411
   training.egi.eu:
     auth:
-      project_id: e51b8a89b30945adbf52a5d568912e4c
+      project_id: e51b8a89b30945adbf52a5d568912e4c
```
Comment (enolfc) [inline sites/NCG-INGRID-PT.yaml:21]:
```suggestion
      project_id: e51b8a89b30945adbf52a5d568912e4c
  
```

**Review by enolfc — APPROVED** [review]
Thanks @mariojmdavid, this can be merged.

**Pull request merged**